main.go

```diff
@@ -42,3 +46,11 @@ func main() {
 	}
 
 }
+
+func formatBuiltWith(Tags string) string {
+	if len(Tags) == 0 {
+  		return ""
+  	}
+
+  	return " built with: " + strings.Replace(Tags, " ", ", ", -1)
+}
```

Comment:
True, but maybe it's even easier to create a comma separated list to split by space and join by comma